Application/EdFi.Ods.WebApi.NetCore/appsettings.Development.json

```
@@ -18,6 +18,22 @@
     "DisableSecurity": false,
     "UseReverseProxyHeaders": false,
     "Years": [],
+    "SandboxTemplate": [
+      {
+        "Key": "PopulatedTemplateScript",
+        "Value": "GrandBend"
+      }
+    ],
+    "DatabaseTemplate": [
+      {
+        "Key": "PopulatedTemplateScript",
+        "Value": "GrandBend"
+      },
+      {
+        "Key": "MinimalTemplateScript",
+        "Value": "EdFiMinimalTemplate"
+      }
+    ],
```

this json looks odd...
1. we should have a separate section that is bound to its own dto
2. the json should like this:
```
{
  "sandboxTemplates" : {
    "minimal" : "EdFiMinimalTemplate",
    "populated": "GrandBend" 
  },
  "apisettings": {}
}
```

---

@tbmeador-dlp do you have any thoughts?

---

based off the original web.config I'd go with
```suggestion
    "databaseTemplate": [
      {
        "PopulatedTemplateScript": "GrandBend",
        "MinimalTemplateScript": "EdFiMinimalTemplate"
      },
    ],
```
But also keep in mind that there are also postgres versions that use different scripts

---

> this json looks odd...
> 
> 1. we should have a separate section that is bound to its own dto
> 2. the json should like this:
> 
> ```
> {
>   "sandboxTemplates" : {
>     "minimal" : "EdFiMinimalTemplate",
>     "populated": "GrandBend" 
>   },
>   "apisettings": {}
> }
> ```

Modified the Json file to above format